parsec/core/fs/remote_loader.py

```diff
@@ -447,21 +447,21 @@ async def load_block(self, access: BlockAccess) -> None:
         # Download
         with translate_backend_cmds_errors():
             rep = await self.backend_cmds.block_read(access.id)
-        if rep["status"] == "not_found":
+        if rep == BlockReadRep.NotFound():
```

Comment:
use `isinstance(rep, BlockReadRep.NotFound)` instead

and brace yourself, [structural matching is avaialabe in Python 3.10](https://peps.python.org/pep-0636/) ^^

Reply:
Are you sure that `BlockReadRep.NotFound` is a type ?

Reply:
if it's not a type, then what `BlockReadRep.NotFound()` does ? ^^

Reply:
It creates an object